### ISSUE TITLE
Error checks for area calculation

### ISSUE THREAD

**Issue by rajeeja** [UXARRAY/uxarray#425]
**Describe the bug**
Area calculation is incorrect?
For mesh `gridfile_fesom = current_path / "meshfiles" / "ugrid" / "fesom" / "fesom.mesh.diag.nc"`

**To Reproduce**
sum of area comes to be 9776.665489773564

**Expected behavior**
Some message and more checks might be needed.

**Comment by Sevans711** [comment]
This seems fine now? On uxarray 2026.6.0, I am getting:
```python
import uxarray as ux
gridpath = ... # path to `"uxarray/test/meshfiles/ugrid/fesom/fesom.mesh.diag.nc"` on my machine
grid = ux.open_grid(gridpath)
area = grid.calculate_total_face_area()  # area in steradians
print(area)
```
Gives `8.378036745330922`. This seems reasonable to me; `grid.plot()` looks like this:

<img width="597" height="294" alt="Image" src="https://github.com/user-attachments/assets/fd459db8-a512-41c0-a43e-5231ecf6c5a7" />

I can certainly believe this grid covers roughly  67% (`== area / (4 * np.pi)`) of the surface.

If this makes sense to you too please mark as closed!

**Comment by rajeeja** [comment]
@Sevans711 Confirmed — the bad value is fixed; the FESOM mesh now gives ~8.38 sr, which is correct. I'll still add the error check this issue asked for: a warning when total face area exceeds the unit-sphere area (4π), so a malformed grid can't return garbage silently. That's in #1577, which will close this issue.

**Comment by Sevans711** [comment]
Clarifying: the error check proposed to be added to #1577 was removed, because it was originally added in a way that does not respect grid.sphere_radius (when not on unit sphere, area can easily be larger than 4π), and in a way that adds inefficiency to every face_areas computation since `face_areas` doesn't otherwise require summing across all faces.

So, what remains to close this issue? I think doing both of the following would be sufficient:
- open a separate PR (sometime after #1577 merges into main), check inside `grid.calculate_total_face_areas()` (which is already doing a sum across all faces) whether the result is too large, and make warning if it is. This check should compute "too large" as "larger than `4 * np.pi * grid.sphere_radius**2`" not just 4π.
- add a regression test for this fesom grid in particular, which will crash if the total area isn't close to ~8.38 sr.
